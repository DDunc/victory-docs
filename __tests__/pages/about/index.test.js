/* global describe, test, expect */
import React from "react";
import { mount } from "enzyme";
import About from "../../../src/pages/about/index";

describe("<About />", () => {
  test("should render correctly.", () => {
    const wrapper = mount(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
