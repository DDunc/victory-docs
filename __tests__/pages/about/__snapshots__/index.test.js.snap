// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render correctly. 1`] = `
<About>
  <div>
    <article
      className="Article"
    >
      <h1
        className="u-noMargin"
      >
        Victory: Charting for React and React Native
      </h1>
      <p>
        <iframe
          frameBorder="0"
          height="30px"
          scrolling="0"
          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=star&count=true&size=large"
          width="160px"
        >
           
        </iframe>
        <iframe
          frameBorder="0"
          height="30px"
          scrolling="0"
          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=watch&count=true&size=large&v=2"
          width="160px"
        >
           
        </iframe>
        <iframe
          frameBorder="0"
          height="30px"
          scrolling="0"
          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=fork&count=true&size=large"
          width="158px"
        >
           
        </iframe>
      </p>
      <p>
        Victory is a set of modular charting components for React and React Native. Victory makes it easy to get started without sacrificing flexibility. Create one of a kind data visualizations with fully customizable styles and behaviors. Victory uses the same API for web and React Native applications for easy cross-platform charting.
      </p>
      <p>
        Victory is helmed by Formidable’s 
        <a
          href="https://github.com/boygirl"
        >
           Lauren Eastridge
        </a>
        .
      </p>
      <p>
        <a
          href="https://github.com/FormidableLabs/victory/graphs/contributors"
        >
          See Victory Contributors
        </a>
      </p>
      <h2
        id="showcase"
      >
        Victory in Use
      </h2>
      <p>
        Victory is used for charting across the web, from publicly-consumed informational graphs to internal tracking and reporting.
      </p>
      <Showcase>
        <div
          className="Showcases"
        >
          <ShowcaseApp
            company="FiveThirtyEight"
            description="FiveThirtyEight uses Victory for data visualizations, including this one about the history of sumo wrestling."
            key="0"
            screenshot={
              Object {
                "alt": "FiveThirtyEight screenshot",
                "src": "test-file-stub",
              }
            }
          >
            <div
              className="Grid Grid--guttersSm Grid--center Grid--full medium-Grid--fit u-textLarge"
            >
              <div
                className="Grid-cell Grid-cell--autoSize u-maxWidth"
              >
                <img
                  alt="FiveThirtyEight screenshot"
                  className="Showcase-img fancyBorder"
                  src="test-file-stub"
                />
              </div>
              <div
                className="Grid-cell"
              >
                <h3
                  className="SubHeading u-textLeft u-noMargin"
                >
                  FiveThirtyEight
                </h3>
                <p>
                  FiveThirtyEight uses Victory for data visualizations, including this one about the history of sumo wrestling.
                </p>
              </div>
            </div>
          </ShowcaseApp>
          <ShowcaseApp
            company="Postmark"
            description="Postmark uses Victory on its status page to let users know how their service is performing."
            key="1"
            screenshot={
              Object {
                "alt": "Postmark screenshot",
                "src": "test-file-stub",
              }
            }
          >
            <div
              className="Grid Grid--guttersSm Grid--center Grid--full medium-Grid--fit u-textLarge"
            >
              <div
                className="Grid-cell Grid-cell--autoSize u-maxWidth"
              >
                <img
                  alt="Postmark screenshot"
                  className="Showcase-img fancyBorder"
                  src="test-file-stub"
                />
              </div>
              <div
                className="Grid-cell"
              >
                <h3
                  className="SubHeading u-textLeft u-noMargin"
                >
                  Postmark
                </h3>
                <p>
                  Postmark uses Victory on its status page to let users know how their service is performing.
                </p>
              </div>
            </div>
          </ShowcaseApp>
          <ShowcaseApp
            company="Tune"
            description="Tune uses Victory in its Marketing Console, a tool for marketers to track all of their mobile data."
            key="2"
            screenshot={
              Object {
                "alt": "Tune screenshot",
                "src": "test-file-stub",
              }
            }
          >
            <div
              className="Grid Grid--guttersSm Grid--center Grid--full medium-Grid--fit u-textLarge"
            >
              <div
                className="Grid-cell Grid-cell--autoSize u-maxWidth"
              >
                <img
                  alt="Tune screenshot"
                  className="Showcase-img fancyBorder"
                  src="test-file-stub"
                />
              </div>
              <div
                className="Grid-cell"
              >
                <h3
                  className="SubHeading u-textLeft u-noMargin"
                >
                  Tune
                </h3>
                <p>
                  Tune uses Victory in its Marketing Console, a tool for marketers to track all of their mobile data.
                </p>
              </div>
            </div>
          </ShowcaseApp>
          <ShowcaseApp
            company="Cumul8"
            description="Cumul8 uses Victory for Cirrus, a SaaS platform for businesses to manage their internal data."
            key="3"
            screenshot={
              Object {
                "alt": "Cumul8 screenshot",
                "src": "test-file-stub",
              }
            }
          >
            <div
              className="Grid Grid--guttersSm Grid--center Grid--full medium-Grid--fit u-textLarge"
            >
              <div
                className="Grid-cell Grid-cell--autoSize u-maxWidth"
              >
                <img
                  alt="Cumul8 screenshot"
                  className="Showcase-img fancyBorder"
                  src="test-file-stub"
                />
              </div>
              <div
                className="Grid-cell"
              >
                <h3
                  className="SubHeading u-textLeft u-noMargin"
                >
                  Cumul8
                </h3>
                <p>
                  Cumul8 uses Victory for Cirrus, a SaaS platform for businesses to manage their internal data.
                </p>
              </div>
            </div>
          </ShowcaseApp>
        </div>
      </Showcase>
      <h1>
        About Formidable
      </h1>
      <p>
        Formidable is a Seattle-based consultancy and development shop, focused on open-source, full-stack JavaScript using React.js and Node.js, and the architecture of large-scale JavaScript applications. We build products for some of the world’s biggest companies, while helping their internal teams develop smart, thoughtful, and scalable systems.
      </p>
    </article>
    <VictoryFooter
      home={false}
    >
      <Footer
        className="victory"
        theme="dark"
        trademark={
          <div
            className="default"
          >
            Victory is a trademark of Formidable Labs, Inc.
          </div>
        }
      >
        <footer
          className="formidableFooter isDark victory"
        >
          <div
            className="formidableFooter-container"
          >
            <div
              className="default"
            >
              <a
                href="https://formidable.com/contact/"
              >
                Contact
              </a>
              <a
                href="https://formidable.com/careers/"
              >
                Careers
              </a>
              <a
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "test-file-stub",
                  }
                }
                href="https://twitter.com/FormidableLabs"
              />
              <a
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "test-file-stub",
                  }
                }
                href="https://github.com/FormidableLabs/"
              />
            </div>
            <div
              className="formidableFooter-logo"
            >
              <a
                className="formidableFooter-logo-link"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "test-file-stub",
                  }
                }
                href="https://formidable.com/"
                target="_blank"
              />
            </div>
            <div
              className="default"
            >
              Victory is a trademark of Formidable Labs, Inc.
            </div>
          </div>
        </footer>
      </Footer>
    </VictoryFooter>
  </div>
</About>
`;
